docs/en/observability/inspect-metric-anomalies.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[inspect-metric-anomalies]]
+= Inspect metric anomalies
+
+When the {anomaly-detect} features of {ml} are enabled,
+you can create {ml} jobs to detect and inspect memory usage
+and network traffic anomalies for hosts and Kubernetes pods.
+
+You can model system memory usage, along with inbound and outbound
+network traffic across hosts or pods. You can detect unusual
+increases in memory usage, and unusually high inbound or outbound traffic
+across hosts or pods.
+
+[ml-jobs-hosts]]
+== Enable {ml} jobs for hosts or Kubernetes pods
+
+Create a {ml} job to detect anomalous memory usage and network traffic automatically.
+
+Once the {ml} jobs are created, these settings can not be changed. You can recreate
+these jobs later. However, any previously detected anomalies are removed.
+
+1. In the side navigation, expand *Observability*, click *Metrics*, and then click *Anomaly detection*.
+2. You’ll be prompted to create a machine learning job for *Hosts* or *Kubernetes Pods*.
+3. Choose a start date for the machine learning analysis.
++
+Machine learning jobs analyze the past four weeks of data and continue to run indefinitely.
+You can specify a different start date, end date, or both, but we recommend that you run jobs
```

Comment:
I cannot specify the end date when creating the job. Is it really possible?

Reply:
Just checked the latest build and you're correct. You can only specify the start date.